scripts/run.sh

```diff
@@ -94,8 +96,8 @@ mkdir -p ${TMPDIR}
 ls -al ${TMPDIR}
 
 # Input file parsing
-EXTENSION=".hepmc3.tree.root"
```

Comment:
EXTENSION starts with `.`

Reply:
i.e. why the change, and are there side effects?

Reply:
in the csv files extensions don't have the "." [rahmans@osgsub01 ~/job_submission_condor]$ head -1 bgplussynrad_10x100_q2_10_100-2024-05-15T14:05-0400.csv
BACKGROUNDS/10x100/merged_lowstat/bgmerged_10000_ep_noradcor_10x100_q2_10_100_run001,hepmc3.tree.root,416,0000

Reply:
Ok, without makes more sense then.